src/app/wizard/step/cluster/component.ts

```diff
@@ -172,6 +184,12 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
     return AdmissionPluginUtils.isPluginEnabled(this.form.get(Controls.AdmissionPlugins), name);
   }
 
+  private _enable(control: Controls, isEnabled: boolean): void {
+    if (isEnabled) {
+      this.form.get(control).setValue(true);
```

Comment:
Can't we just use `this.form.get(control).setValue(isEnabled);`? If `isEnabled` is false then it will stay false, if it is true then it will be set to true.

Reply:
Is there any reason for avoiding the `setValue()` call?